doc/user-guide/dask.rst

```diff
@@ -296,6 +296,12 @@ loaded into Dask or not:
 Automatic parallelization with ``apply_ufunc`` and ``map_blocks``
 -----------------------------------------------------------------
 
+.. tip::
+
+   Some problems can become embarassingly parallel and thus easy to parallelize
+   automatically by rechunking to a frequency, e.g. ``ds.chunk(time=TimeResampler("YE"))``.
```

Comment:
I think currently the only way to import / use `TimeResampler` from the public API is via either:
```
>>> import xarray
>>> xarray.groupers.TimeResampler
```
or
```
>>> from xarray import groupers
>>> groupers.TimeResampler
```
In other words it does not seem possible to import the name `TimeResampler` alone unless one does the following (which I think we would discourage):
```
>>> from xarray.core.groupers import TimeResampler
```
We should probably do something to address this or update the documentation.

Reply:
Nice catch, how would we enable `from xarray.groupres import TimeResampler`? Moving the file up from `core/`?

Reply:
Fixed by #9289

Reply:
Great, thanks!